cellmlmanip/model.py

```diff
@@ -536,8 +550,8 @@ def get_ontology_terms_by_variable(self, variable, namespace_uri=None):
     def get_definition(self, variable):
         """Get the equation (if any) defining the given variable.
 
-        :param variable: The variable to look up (as a class:`VariableDummy`. If this appears as the LHS of a straight
-            assignment, or the state variable in an ODE, the corresponding equation will be returned.
+        :param variable: The variable to look up (as a :class:`VariableDummy`). If this appears as the LHS of a
```

Comment:
Did we not have a linewidth of 120 or something?

Reply:
That was just me getting scared when my editor told me I was on column 121...